packages/core/core.ts

```diff
@@ -39,15 +40,18 @@ function initFuzzing(options: Options): fuzzer.FuzzFn {
 	globalThis.Fuzzer = fuzzer.fuzzer;
 	//TODO make sure that all sanitizers are registered at this point
 	globalThis.HookManager = hooking.hookManager;
+	// load each custom hook file
+	options.customHooks.forEach((customHook) => {
+		const fuzzFn = importModule(customHook);
```

Comment:
No need to assign the module to a local variable.

Reply:
Updated and moved into 17c51ef.